dev-server/containers/MainView/index.js

```diff
@@ -61,6 +61,17 @@ const MainView = connect((state, props) => {
   if (messageTab) {
     messageTab.noticeCounts = props.messageStore.unreadCounts;
   }
+  if (props.rolesAndPermissions.permissions &&
+      props.rolesAndPermissions.permissions.OutboundSMS !== true &&
+      props.rolesAndPermissions.permissions.InternalSMS !== true
```

Comment:
Do you mean when a user has SMS/MMS/pager permission and no readmessages permission,hide messages page but show composeText page？